usr/libexec/greenboot/greenboot

```diff
@@ -7,6 +7,7 @@ script_runner () {
   local scripts_dir=$1; shift
   local mode=$1; shift
   local start_msg=$1; shift
+  local required_hc_failed="false"
```

Comment:
I don't think quotation marks are necessary here

Reply:
You're right. Pushing changes...